tensorflow_datasets/core/visualization/show_examples.py

```diff
@@ -56,9 +56,7 @@ def show_examples(ds_info, ds, **options_kwargs):
   """
   for visualizer in _ALL_VISUALIZERS:
     if visualizer.match(ds_info):
-      visualizer.show(ds_info, ds, **options_kwargs)
-      break
```

Comment:
Why remove the break ?

Reply:
Because I am returning `visualizer.show(ds_info, ds, **options_kwargs)` this changes I have to do because in `generate_visualization.py` I am saving fig . See PR #1885 have these changes separately